lib/web_ui/test/golden_tests/engine/backdrop_filter_golden_test.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
The implementation is sensitive to the surface lifecycle. We should have tests that move the surface around (both as retained and as updated) and check that the style is applied correctly.

Reply:
Added test that moves clipRect.